frontend/src/components/OrderPage.js

```diff
@@ -761,6 +812,43 @@ class OrderPage extends Component {
     )
   }
 
+  handleCloseWeblnDialog = () => {
+    this.setState({openWeblnDialog: false});
+  }
+
+  weblnDialog =() =>{
+    const { t } = this.props;
+
+    return(
+      <Dialog
+        open={this.state.openWeblnDialog}
+        onClose={this.handleCloseWeblnDialog}
+        aria-labelledby="webln-dialog-title"
+        aria-describedby="webln-dialog-description"
+      >
+        <DialogTitle id="webln-dialog-title">
+          {t("WebLN")}
+        </DialogTitle>
+        <DialogContent>
+          <DialogContentText id="webln-dialog-description">
+            {this.state.waitingWebln ? 
+              <>
+                <CircularProgress size={16} thickness={5} style={{ marginRight: 10 }}/>
+                {this.state.is_buyer ? t("Invoice not received, please check your WebLN wallet.") : t("Payment not received, please check your WebLN wallet.")} 
+              </> 
+              : <>
+                  {t("You can close now your WebLN wallet popup.")}
```

Comment:
To highlight the change in the dialog, once the CircularProgress spinner dissapears we could introduce a green checkmark
```javascript
import CheckIcon from '@mui/icons-material/Check';
...
<CheckIcon color="success"/>
{t("You can close now your WebLN wallet popup.")}
...
```